Static Analysis Results Interchange Format (SARIF).html

```diff
@@ -2431,25 +2431,32 @@ <h2 id="result-management-profile"  data-line="34;document\ResultManagementProfi
 </p>
 <p class="indent" data-line="46;document\AnnexC.mdk:13"><span data-line="46;document\AnnexC.mdk:13"></span>Converters should populate those elements of the SARIF format
 for which a direct equivalent exists in the input data.
-If the input data includes information for which there is no SARIF equivalent,
+</p>
+<p class="indent" data-line="46;document\AnnexC.mdk:16"><span data-line="46;document\AnnexC.mdk:16"></span>If the input data includes information for which there is no SARIF equivalent,
 converters may use it to populate the various property bags and tag lists defined by the SARIF format,
 or they may simply omit it from the output.
-If the input data does not include an equivalent for any SARIF element,
-the converter should <span data-line="46;document\AnnexC.mdk:19"></span><em>not</em><span data-line="46;document\AnnexC.mdk:19"></span> attempt to synthesize that element.
+When populating a property bag with such information,
+converters should use a property name that matches the name of that piece of information in the
+native tool format (assuming of course that that name is a valid JSON property name),
```

Comment:
do you have guidance for the condition where a property name is not a valid JSON property name?